examples/vector_databases/neon/neon-postgres-vector-search-pgvector.ipynb

```diff
@@ -0,0 +1,538 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Vector similarity search using Neon Postgres with pgvector and OpenAI\n",
+    "\n",
+    "This notebook guides you through using [Neon Serverless Postgres](https://neon.tech/) as a vector database for OpenAI embeddings. It demonstrates how to:\n",
+    "\n",
+    "1. Use embeddings created by OpenAI API\n",
```

Comment:
could add periods here for consistency with the other lists

Reply:
Thank you, @simonpfish. Added periods, as suggested.